docs/testing/testplan.md

```diff
@@ -0,0 +1,355 @@
+3. When the extension facilitates paying a monetized web page, check how payments get distributed between the receiving wallets on the page.
+
+## Dependencies
+
+### Digital Wallets
+
+The main prerequisite for sending or receiving Web Monetization payments is to have a Web Monetization-enabled digital wallet. Web Monetization-enabled digital wallets are provided by licensed service providers. A wallet allows a user to send or receive Web Monetization payments. The wallet providers are regulated by the laws of the countries in which they operate.
+
+Below is a list of the available wallet providers if you want to use real money:
+
+- [Fynbos](https://wallet.fynbos.app/wallet)
+- [GateHub](https://gatehub.net/mobile)
+
+Wallet availability in a specific country, or availability in particular currency depends on the wallet provider.  
+Learn more about the providers for [Web Monetization compatible wallets here](https://webmonetization.org/docs/resources/op-wallets/).
+
+#### Test Digital Wallet
+
+You have the option to use “play” money from the [Interledger Test Wallet application](https://rafiki.money/).  
+Here you can set up an account that is enabled for Web Monetization, and other Interledger functionality, without involving real money.
+
+You can learn more about the [Interledger Test Wallet here](https://rafiki.dev/integration/playground/testnet/).
+
+## Testing
+
+### Environment and Software
+
+The resources and tools that you will need to test Web Monetization depend on your desired role, and what you want to test.
+
+We’ve identified three roles you can play when using Web Monetization:
+
+- Website visitor: Wants to pay websites that are web monetized
+- Website owner: Wants to receive Web Monetization payments
+- Web developer: Wants to explore Web Monetization functionality.
+
+This test plan only focuses on the perspective of a website visitor.
+
+| Role              | Hardware                    | Digital wallet                                                             | Environment & software                                                                                                    |
+| :---------------- | :-------------------------- | :------------------------------------------------------------------------- | :------------------------------------------------------------------------------------------------------------------------ |
+| A website visitor | Any PC with internet access | A Web Monetization enabled digital wallet from which payments will be made | <ul><li>Any operating system (OS)</li> <li>A chromium-based Web browser (e.g. Chrome, Edge, Brave) or Firefox </li> </ul> |
+
+### Reporting Issues
+
+1. Use any screen recording application to record the steps and results of your testing.
+2. Log bugs or issues, and monitor their status and resolution on GitHub [link](https://github.com/interledger/web-monetization-extension/issues).
+
+### Prerequisites
+
+Before you start testing, sign-up for a wallet:
```

Comment:
Should we mention that the prereqs are different for the Test wallet?

Reply:
Cool 👍🏾